src/RustyObjectStore.jl

```diff
@@ -30,20 +36,31 @@ Global configuration for the object store requests.
 # Keywords
 $TYPEDFIELDS
 """
-@kwdef struct ObjectStoreConfig
-    "The maximum number of times to retry a request."
-    max_retries::Culonglong
-    "The number of seconds from the initial request after which no further retries will be attempted."
-    retry_timeout_sec::Culonglong
+@kwdef struct StaticConfig
+    "The number of worker threads for the native pool (0 == auto)"
+    n_threads::Culonglong
+    "The maximum capacity for the client cache"
+    cache_capacity::Culonglong
+    "The time-to-live in seconds for entries in the client cache"
+    cache_ttl_secs::Culonglong
```

Comment:
I'm assuming 0 here will disable ttl and not have it instantly expired?

Reply:
Well, I would have hoped too, but the way I did it on the Rust side will actually make it expire instantly. Sadly changing this requires a new release of object_store_ffi_jl. We could leave that as an issue on the Rust side so we remember to make this change for the next release. Do you think I should place a guard here, in the mean time, to set a huge default duration in the case zero is provided?

Reply:
So the fix on the rust side is if we are passed 0 then we just don't set the config option on the cache, right? I can include that on my PR with the multipart operations, just so we don't keep bouncing around with PRs for version bumps. In that case I don't think you need to change anything here.

Reply:
Yes, exactly like the `n_threads` config in the runtime. Thanks!